packages/fhir-converter/src/templates/cda/Utils/GeneratePractitionerId.hbs

```diff
@@ -31,7 +31,7 @@
             {{#if (and FirstId.root FirstId.extension)}}
                 "{{generateUUID (concat (toString FirstId.root) '|' (toString FirstId.extension))}}"
             {{else if ../obj.assignedPerson.name}}
-                "{{generateUUID (concat ../obj.assignedPerson.name ../obj.addr ../obj.telecom)}}"
```

Comment:
basically, the issue here was that it was comparing `[object Object]` strings instead of the actual contents of the corresponding objects